### ISSUE TITLE
Getting Failed with forward port error while working on Android Chrome browser automation test.

### ISSUE THREAD

**Issue by AmitKambleOnset** [appium/appium#13235]
## The problem
Working on Android chrome browser automation , since i upgraded the Android studio to latest version i am getting this error - ERROR unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.

## Environment
* Appium version :-  v1.14.1
* Last Appium version that did not exhibit the issue (if applicable):  v1.14.1
* Desktop OS/version used to run Appium: Windows 10
* Node.js version (unless using Appium.app|exe):10:16:3
* Mobile platform/version under test: Android 8.0.0
* Real device or emulator/simulator:Galaxy S9
* Appium CLI or Appium.app|exe:Appium CLI

## Details
Issue is appearing since Android studio is upgraded from 3.4.2 to 3.5 version.

## Link to Appium logs

[Appium] Welcome to Appium v1.14.1
[Appium] Appium REST http interface listener started on 0.0.0.0:4723
[HTTP] --> POST /wd/hub/session
[HTTP] {"desiredCapabilities":{"noReset":true,"platformVersion":"8.0.0","showChromedriverLog":true,"automationName":"UiAutomator2","autoGrantPermissions":true,"browserName":"Chrome","unicodeKeyboard":true,"platformName":"Android","deviceName":"Galaxy S8","resetKeyboard":true,"autoAcceptAlerts":true},"capabilities":{"firstMatch":[{"appium:autoAcceptAlerts":true,"appium:autoGrantPermissions":true,"appium:automationName":"UiAutomator2","browserName":"Chrome","appium:deviceName":"Galaxy S8","appium:noReset":true,"platformName":"android","appium:platformVersion":"8.0.0","appium:resetKeyboard":true,"showChromedriverLog":true,"appium:unicodeKeyboard":true}]}}
[debug] [W3C] Calling AppiumDriver.createSession() with args: [{"noReset":true,"platformVersion":"8.0.0","showChromedriverLog":true,"automationName":"UiAutomator2","autoGrantPermissions":true,"browserName":"Chrome","unicodeKeyboard":true,"platformName":"Android","deviceName":"Galaxy S8","resetKeyboard":true,"autoAcceptAlerts":true},null,{"firstMatch":[{"appium:autoAcceptAlerts":true,"appium:autoGrantPermissions":true,"appium:automationName":"UiAutomator2","browserName":"Chrome","appium:deviceName":"Galaxy S8","appium:noReset":true,"platformName":"android","appium:platformVersion":"8.0.0","appium:resetKeyboard":true,"showChromedriverLog":true,"appium:unicodeKeyboard":true}]}]
[debug] [BaseDriver] Event 'newSessionRequested' logged at 1568410007447 (17:26:47 GMT-0400 (Eastern Daylight Time))
[BaseDriver] The capabilities ["showChromedriverLog"] are not standard capabilities and should have an extension prefix
[Appium] Appium v1.14.1 creating new AndroidUiautomator2Driver (v1.33.1) session
[Appium] Capabilities:
[Appium]   browserName: Chrome
[Appium]   platformName: android
[Appium]   showChromedriverLog: true
[Appium]   autoAcceptAlerts: true
[Appium]   autoGrantPermissions: true
[Appium]   automationName: UiAutomator2
[Appium]   deviceName: Galaxy S8
[Appium]   noReset: true
[Appium]   platformVersion: 8.0.0
[Appium]   resetKeyboard: true
[Appium]   unicodeKeyboard: true
[debug] [BaseDriver] W3C capabilities {"alwaysMatch":{"browserNam... and MJSONWP desired capabilities {"noReset":true,"platformVe... were provided
[debug] [BaseDriver] Creating session with W3C capabilities: {"alwaysMatch":{"browserNam...
[BaseDriver] The following capabilities were provided, but are not recognized by appium: autoAcceptAlerts.
[BaseDriver] Session created with session id: c22dc87d-14af-4434-8d25-d6ba4375490f
[UiAutomator2] We're going to run a Chrome-based session
[UiAutomator2] Chrome-type package and activity are com.android.chrome and com.google.android.apps.chrome.Main
[ADB] Found 1 'build-tools' folders under 'C:\Users\akamble\AppData\Local\Android\Sdk' (newest first):
[ADB]     C:/Users/akamble/AppData/Local/Android/Sdk/build-tools/29.0.2
[ADB] Using 'adb.exe' from 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe'
[AndroidDriver] Retrieving device list
[debug] [ADB] Trying to find a connected android device
[debug] [ADB] Getting connected devices...
[debug] [ADB] 1 device(s) connected
[AndroidDriver] Looking for a device with Android '8.0.0'
[debug] [ADB] Setting device id to 988e9031364837354b
[ADB] Getting device platform version
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell getprop ro.build.version.release'
[debug] [ADB] Current device property 'ro.build.version.release': 8.0.0
[AndroidDriver] Using device: 988e9031364837354b
[ADB] Using 'adb.exe' from 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe'
[debug] [ADB] Setting device id to 988e9031364837354b
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell getprop ro.build.version.sdk'
[debug] [ADB] Current device property 'ro.build.version.sdk': 26
[debug] [ADB] Device API level: 26
[AndroidDriver] No app sent in, not parsing package/activity
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b wait-for-device'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell echo ping'
[debug] [AndroidDriver] Pushing settings apk to device...
[debug] [ADB] Getting install status for io.appium.settings
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys package io.appium.settings'
[debug] [ADB] 'io.appium.settings' is installed
[debug] [ADB] Getting package info for 'io.appium.settings'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys package io.appium.settings'
[ADB] Using 'aapt.exe' from 'C:\Users\akamble\AppData\Local\Android\Sdk\build-tools\29.0.2\aapt.exe'
[debug] [ADB] The version name of the installed 'io.appium.settings' is greater or equal to the application version name ('2.14.0' >= '2.14.0')
[debug] [ADB] There is no need to install/upgrade 'C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk'
[debug] [ADB] Getting IDs of all 'io.appium.settings' processes
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell 'pgrep --help; echo $?''
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell pgrep \^appium\\.settings\$'
[debug] [AndroidDriver] io.appium.settings is already running. There is no need to reset its permissions.
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell appops set io.appium.settings android\:mock_location allow'
[debug] [Logcat] Starting logcat capture
[debug] [AndroidDriver] Enabling Unicode keyboard support
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell settings get secure default_input_method'
[debug] [AndroidDriver] Unsetting previous IME com.sec.android.inputmethod/.SamsungKeypad
[debug] [AndroidDriver] Setting IME to 'io.appium.settings/.UnicodeIME'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell ime enable io.appium.settings/.UnicodeIME'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell ime set io.appium.settings/.UnicodeIME'
[debug] [ADB] Getting install status for io.appium.uiautomator2.server
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys package io.appium.uiautomator2.server'
[debug] [ADB] 'io.appium.uiautomator2.server' is installed
[debug] [ADB] Getting package info for 'io.appium.uiautomator2.server'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys package io.appium.uiautomator2.server'
[debug] [ADB] The version name of the installed 'io.appium.uiautomator2.server' is greater or equal to the application version name ('3.5.1' >= '3.5.1')
[debug] [UiAutomator2] io.appium.uiautomator2.server installation state: sameVersionInstalled
[debug] [ADB] Checking app cert for C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v3.5.1.apk
[ADB] Using 'apksigner.bat' from 'C:\Users\akamble\AppData\Local\Android\Sdk\build-tools\29.0.2\apksigner.bat'
[debug] [ADB] Starting 'C:\Users\akamble\AppData\Local\Android\Sdk\build-tools\29.0.2\apksigner.bat' with args '["verify","--print-certs","C:\\Users\\swdl\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-uiautomator2-server\\apks\\appium-uiautomator2-server-v3.5.1.apk"]'
[debug] [ADB] apksigner stdout: Signer #1 certificate DN: EMAILADDRESS=android@android.com, CN=Android, OU=Android, O=Android, L=Mountain View, ST=California, C=US
[debug] [ADB] Signer #1 certificate SHA-256 digest: a40da80a59d170caa950cf15c18c454d47a39b26989d8b640ecd745ba71bf5dc
[debug] [ADB] Signer #1 certificate SHA-1 digest: 61ed377e85d386a8dfee6b864bd85b0bfaa5af81
[debug] [ADB] Signer #1 certificate MD5 digest: e89b158e4bcf988ebd09eb83f5378e87
[debug] [ADB]
[debug] [ADB] 'C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v3.5.1.apk' is already signed.
[debug] [ADB] Getting install status for io.appium.uiautomator2.server.test
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys package io.appium.uiautomator2.server.test'
[debug] [ADB] 'io.appium.uiautomator2.server.test' is installed
[debug] [ADB] Checking app cert for C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk
[debug] [ADB] Starting 'C:\Users\akamble\AppData\Local\Android\Sdk\build-tools\29.0.2\apksigner.bat' with args '["verify","--print-certs","C:\\Users\\swdl\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\appium-uiautomator2-server\\apks\\appium-uiautomator2-server-debug-androidTest.apk"]'
[debug] [ADB] apksigner stdout: Signer #1 certificate DN: EMAILADDRESS=android@android.com, CN=Android, OU=Android, O=Android, L=Mountain View, ST=California, C=US
[debug] [ADB] Signer #1 certificate SHA-256 digest: a40da80a59d170caa950cf15c18c454d47a39b26989d8b640ecd745ba71bf5dc
[debug] [ADB] Signer #1 certificate SHA-1 digest: 61ed377e85d386a8dfee6b864bd85b0bfaa5af81
[debug] [ADB] Signer #1 certificate MD5 digest: e89b158e4bcf988ebd09eb83f5378e87
[debug] [ADB]
[debug] [ADB] 'C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk' is already signed.
[UiAutomator2] Server packages are not going to be (re)installed
[debug] [UiAutomator2] Waiting up to 30000ms for services to be available
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell pm list instrumentation'
[debug] [UiAutomator2] Instrumentation target 'io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner' is available
[debug] [UiAutomator2] Forwarding UiAutomator2 Server port 6790 to 8200
[debug] [ADB] Forwarding system: 8200 to device: 6790
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b forward tcp\:8200 tcp\:6790'
[debug] [UiAutomator2] No app capability. Assuming it is already on the device
[debug] [UiAutomator2] Performing shallow cleanup of automation leftovers
[debug] [UiAutomator2] No obsolete sessions have been detected (Error: socket hang up)
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell am force-stop io.appium.uiautomator2.server.test'
[UiAutomator2] Starting UIAutomator2 server 3.5.1
[UiAutomator2] Using UIAutomator2 server from 'C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v3.5.1.apk' and test from 'C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[UiAutomator2] Waiting up to 30000ms for UiAutomator2 to be online...
[debug] [ADB] Creating ADB subprocess with args: ["-P",5037,"-s","988e9031364837354b","shell","am","instrument","-w","io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner"]
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://localhost:8200/wd/hub/status] with no body
[WD Proxy] Got an unexpected response: {"code":"ECONNRESET"}
[debug] [Instrumentation] io.appium.uiautomator2.server.test.AppiumUiAutomator2Server:
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://localhost:8200/wd/hub/status] with no body
[WD Proxy] Got an unexpected response: {"code":"ECONNRESET"}
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://localhost:8200/wd/hub/status] with no body
[debug] [WD Proxy] Got response with status 200: "{\"sessionId\":\"None\",\"status\":0,\"value\":\"Status Invoked\"}"
[debug] [WD Proxy] Determined that the downstream protocol for proxy is MJSONWP
[debug] [UiAutomator2] The initialization of the instrumentation process took 3281ms
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://localhost:8200/wd/hub/session] with body: {"desiredCapabilities":{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"browserName":"Chrome","platformName":"android","showChromedriverLog":true,"autoAcceptAlerts":true,"autoGrantPermissions":true,"automationName":"UiAutomator2","deviceName":"Galaxy S8","noReset":true,"platformVersion":"8.0.0","resetKeyboard":true,"unicodeKeyboard":true},"browserName":"Chrome","platformName":"android","showChromedriverLog":true,"autoAcceptAlerts":true,"autoGrantPermissions":true,"automationName":"UiAutomator2","deviceName":"988e9031364837354b","noReset":true,"platformVersion":"8.0.0","resetKeyboard":true,"unicodeKeyboard":true,"appPackage":"com.android.chrome","appActivity":"com.google.android.apps.chrome.Main","deviceUDID":"988e9031364837354b"}}
[debug] [WD Proxy] Got response with status 200: {"sessionId":"7615fb84-3cf9-4571-a0c0-ee01f6226621","status":0,"value":"Created Session"}
[debug] [WD Proxy] Proxying [GET /appium/device/info] to [GET http://localhost:8200/wd/hub/session/7615fb84-3cf9-4571-a0c0-ee01f6226621/appium/device/info] with no body
[debug] [WD Proxy] Got response with status 200: "{\"sessionId\":\"7615fb84-3cf9-4571-a0c0-ee01f6226621\",\"status\":0,\"value\":{\"androidId\":\"3b24cc9662946d49\",\"manufacturer\":\"samsung\",\"model\":\"SM-G950U1\",\"brand\":\"samsung\",\"apiVersion\":\"26\",\"platformVersion\":\"8.0.0\",\"carrierName\":\"\",\"realDisplaySize\":\"1080x2220\",\"displayDensity\":480,\"networks\":[{\"type\":13,\"typeName\":\"WIFI_P2P\",\"subtype\":0,\"subtypeName\":\"\",\"isConnected\":false,\"detailedState\":\"DISCONNECTED\",\"state\":\"DISCONNECTED\",\"extraInfo\":null,\"isAvailable\":true,\"isFailover\":false,\"isRoaming\":false,\"capabilities\":{\"transportTypes\":\"NET_CAPABILITY_SUPL\",\"networkCapabilities\":\"\",\"linkUpstreamBandwidthKbps\":1048576,\"linkDownBandwidthKbps\":1048576,\"signalStrength\":-2147483648,\"networkSpecifier\":null,\"SSID\":null}},{\"type\":1,\"typeName\":\"WIFI\",\"subtype\":0,\"subtypeName\":\"\",\"isConnected\":true,\"detailedState\":\"CONNECTED\",\"state\":\"CONNECTED\",\"extraInfo\":\"\\\"Welcome To Onset\\\"\",\"isAvailable\":true,\"...
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys window'
[AndroidDriver] Screen already unlocked, doing nothing
[AndroidDriver] Starting a chrome-based browser session
[debug] [AndroidDriver] A port was not given, using random free port: 8000
[debug] [AndroidDriver] Before starting chromedriver, androidPackage is com.android.chrome
[debug] [Chromedriver] Changed state to 'starting'
[debug] [Chromedriver] Found 1 executable in 'C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\chromedriver\win'
[debug] [Chromedriver] The following Chromedriver executables were found:
[debug] [Chromedriver]     C:/Users/swdl/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe (minimum Chrome version '69.0.3497')
[debug] [ADB] Getting package info for 'com.android.chrome'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell dumpsys package com.android.chrome'
[debug] [Chromedriver] Found Chrome bundle 'com.android.chrome' version '77.0.3865'
[debug] [Chromedriver] Found 1 Chromedriver executable capable of automating Chrome '77.0.3865'.
[debug] [Chromedriver] Choosing the most recent, 'C:/Users/swdl/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe'.
[debug] [Chromedriver] If a specific version is required, specify it with the `chromedriverExecutable`desired capability.
[Chromedriver] Set chromedriver binary as: C:/Users/swdl/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe
[debug] [Chromedriver] Killing any old chromedrivers, running: wmic process where "commandline like '%chromedriver.exe%--port=8000%'" delete
[debug] [Chromedriver] Successfully cleaned up old chromedrivers
[debug] [Chromedriver] Cleaning any old adb forwarded port socket connections
[debug] [ADB] List forwarding ports
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b forward --list'
[Chromedriver] Spawning chromedriver with: C:/Users/swdl/AppData/Roaming/npm/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe --url-base=wd/hub --port=8000 --adb-port=5037 --verbose
[debug] [Chromedriver] Chromedriver version: '2.44.609538'
[debug] [Chromedriver] [STDOUT] Starting ChromeDriver 2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90) on port 8000
[debug] [Chromedriver] [STDOUT] Only local connections are allowed.
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8000/wd/hub/status] with no body
[debug] [WD Proxy] Got response with status 200: "{\"sessionId\":\"\",\"status\":0,\"value\":{\"build\":{\"version\":\"alpha\"},\"os\":{\"arch\":\"x86_64\",\"name\":\"Windows NT\",\"version\":\"10.0.17763\"}}}"
[debug] [WD Proxy] Determined that the downstream protocol for proxy is MJSONWP
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.android.chrome","androidDeviceSerial":"988e9031364837354b"},"loggingPrefs":{"browser":"ALL"}}}
[Chromedriver] [STDERR] [1568410024.458][INFO]: [9711d8cc99345cc7256e9bda4cf78a6a] COMMAND InitSession {
[Chromedriver] [STDERR]    "desiredCapabilities": {
[Chromedriver] [STDERR]       "chromeOptions": {
[Chromedriver] [STDERR]          "androidDeviceSerial": "988e9031364837354b",
[Chromedriver] [STDERR]          "androidPackage": "com.android.chrome"
[Chromedriver] [STDERR]       },
[Chromedriver] [STDERR]       "loggingPrefs": {
[Chromedriver] [STDERR]          "browser": "ALL"
[Chromedriver] [STDERR]       }
[Chromedriver] [STDERR]    }
[Chromedriver] [STDERR] }
[Chromedriver] [STDERR] [1568410024.458][DEBUG]: Sending adb command: host:devices
[Chromedriver] [STDERR] [1568410024.461][DEBUG]: Received adb response: 988e9031364837354b      device
[Chromedriver] [STDERR] [1568410024.461][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm path com.android.chrome
[Chromedriver] [STDERR] [1568410024.943][DEBUG]: Received adb response: package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/base.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.config.en.apk
[Chromedriver] [STDERR] [1568410024.943][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am set-debug-app --persistent com.android.chrome
[Chromedriver] [STDERR] [1568410025.037][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410025.037][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm clear com.android.chrome
[Chromedriver] [STDERR] [1568410025.680][DEBUG]: Received adb response: Success
[Chromedriver] [STDERR] [1568410025.680][DEBUG]: Sending command line file: /data/local/tmp/chrome-command-line
[Chromedriver] [STDERR] [1568410025.689][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am start -W -n com.android.chrome/com.google.android.apps.chrome.Main -d data:,
[Chromedriver] [STDERR] [1568410026.317][DEBUG]: Received adb response: Starting: Intent { dat=data:, cmp=com.android.chrome/com.google.android.apps.chrome.Main }
[Chromedriver] [STDERR] Status: ok
[Chromedriver] [STDERR] Activity: com.android.chrome/com.google.android.apps.chrome.Main
[Chromedriver] [STDERR] ThisTime: 541
[Chromedriver] [STDERR] TotalTime: 541
[Chromedriver] [STDERR] WaitTime: 566
[Chromedriver] [STDERR] Complete
[Chromedriver] [STDERR] [1568410026.318][DEBUG]: Sending adb command: host-serial:988e9031364837354b:forward:tcp:0;localabstract:chrome_devtools_remote
[Chromedriver] [STDERR] [1568410026.402][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am force-stop com.android.chrome
[Chromedriver] [STDERR] [1568410026.597][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410026.597][INFO]: [9711d8cc99345cc7256e9bda4cf78a6a] RESPONSE InitSession ERROR unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver] [STDERR] [1568410026.597][DEBUG]: Log type 'driver' lost 0 entries on destruction
[Chromedriver] [STDERR] [1568410026.597][DEBUG]: Log type 'browser' lost 0 entries on destruction
[debug] [WD Proxy] Got response with status 200: {"sessionId":"9711d8cc99345cc7256e9bda4cf78a6a","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"9711d8cc99345cc7256e9bda4cf78a6a","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),pl...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver]   (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.android.chrome","androidDeviceSerial":"988e9031364837354b"},"loggingPrefs":{"browser":"ALL"}}}
[Chromedriver] [STDERR] [1568410026.886][INFO]: [a6ef2b6e0b4b32283d2681253111c39c] COMMAND InitSession {
[Chromedriver] [STDERR]    "desiredCapabilities": {
[Chromedriver] [STDERR]       "chromeOptions": {
[Chromedriver] [STDERR]          "androidDeviceSerial": "988e9031364837354b",
[Chromedriver] [STDERR]          "androidPackage": "com.android.chrome"
[Chromedriver] [STDERR]       },
[Chromedriver] [STDERR]       "loggingPrefs": {
[Chromedriver] [STDERR]          "browser": "ALL"
[Chromedriver] [STDERR]       }
[Chromedriver] [STDERR]    }
[Chromedriver] [STDERR] }
[Chromedriver] [STDERR] [1568410026.886][DEBUG]: Sending adb command: host:devices
[Chromedriver] [STDERR] [1568410026.887][DEBUG]: Received adb response: 988e9031364837354b      device
[Chromedriver] [STDERR] [1568410026.887][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm path com.android.chrome
[Chromedriver] [STDERR] [1568410027.323][DEBUG]: Received adb response: package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/base.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.config.en.apk
[Chromedriver] [STDERR] [1568410027.323][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am set-debug-app --persistent com.android.chrome
[Chromedriver] [STDERR] [1568410027.420][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410027.420][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm clear com.android.chrome
[Chromedriver] [STDERR] [1568410027.916][DEBUG]: Received adb response: Success
[Chromedriver] [STDERR] [1568410027.916][DEBUG]: Sending command line file: /data/local/tmp/chrome-command-line
[Chromedriver] [STDERR] [1568410027.922][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am start -W -n com.android.chrome/com.google.android.apps.chrome.Main -d data:,
[Chromedriver] [STDERR] [1568410028.706][DEBUG]: Received adb response: Starting: Intent { dat=data:, cmp=com.android.chrome/com.google.android.apps.chrome.Main }
[Chromedriver] [STDERR] Status: ok
[Chromedriver] [STDERR] Activity: com.android.chrome/com.google.android.apps.chrome.Main
[Chromedriver] [STDERR] ThisTime: 673
[Chromedriver] [STDERR] TotalTime: 673
[Chromedriver] [STDERR] WaitTime: 712
[Chromedriver] [STDERR] Complete
[Chromedriver] [STDERR] [1568410028.707][DEBUG]: Sending adb command: host-serial:988e9031364837354b:forward:tcp:0;localabstract:chrome_devtools_remote
[Chromedriver] [STDERR] [1568410028.708][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am force-stop com.android.chrome
[Chromedriver] [STDERR] [1568410028.805][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410028.806][INFO]: [a6ef2b6e0b4b32283d2681253111c39c] RESPONSE InitSession ERROR unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver] [STDERR] [1568410028.806][DEBUG]: Log type 'driver' lost 0 entries on destruction
[Chromedriver] [STDERR] [1568410028.806][DEBUG]: Log type 'browser' lost 0 entries on destruction
[debug] [WD Proxy] Got response with status 200: {"sessionId":"a6ef2b6e0b4b32283d2681253111c39c","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"a6ef2b6e0b4b32283d2681253111c39c","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),pl...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver]   (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.android.chrome","androidDeviceSerial":"988e9031364837354b"},"loggingPrefs":{"browser":"ALL"}}}
[Chromedriver] [STDERR] [1568410029.092][INFO]: [366a898d88c48a013b901f4cd637640a] COMMAND InitSession {
[Chromedriver] [STDERR]    "desiredCapabilities": {
[Chromedriver] [STDERR]       "chromeOptions": {
[Chromedriver] [STDERR]          "androidDeviceSerial": "988e9031364837354b",
[Chromedriver] [STDERR]          "androidPackage": "com.android.chrome"
[Chromedriver] [STDERR]       },
[Chromedriver] [STDERR]       "loggingPrefs": {
[Chromedriver] [STDERR]          "browser": "ALL"
[Chromedriver] [STDERR]       }
[Chromedriver] [STDERR]    }
[Chromedriver] [STDERR] }
[Chromedriver] [STDERR] [1568410029.092][DEBUG]: Sending adb command: host:devices
[Chromedriver] [STDERR] [1568410029.094][DEBUG]: Received adb response: 988e9031364837354b      device
[Chromedriver] [STDERR] [1568410029.094][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm path com.android.chrome
[Chromedriver] [STDERR] [1568410029.529][DEBUG]: Received adb response: package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/base.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.config.en.apk
[Chromedriver] [STDERR] [1568410029.529][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am set-debug-app --persistent com.android.chrome
[Chromedriver] [STDERR] [1568410029.590][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410029.590][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm clear com.android.chrome
[Chromedriver] [STDERR] [1568410030.045][DEBUG]: Received adb response: Success
[Chromedriver] [STDERR] [1568410030.045][DEBUG]: Sending command line file: /data/local/tmp/chrome-command-line
[Chromedriver] [STDERR] [1568410030.052][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am start -W -n com.android.chrome/com.google.android.apps.chrome.Main -d data:,
[Chromedriver] [STDERR] [1568410030.671][DEBUG]: Received adb response: Starting: Intent { dat=data:, cmp=com.android.chrome/com.google.android.apps.chrome.Main }
[Chromedriver] [STDERR] Status: ok
[Chromedriver] [STDERR] Activity: com.android.chrome/com.google.android.apps.chrome.Main
[Chromedriver] [STDERR] ThisTime: 515
[Chromedriver] [STDERR] TotalTime: 515
[Chromedriver] [STDERR] WaitTime: 554
[Chromedriver] [STDERR] Complete
[Chromedriver] [STDERR] [1568410030.671][DEBUG]: Sending adb command: host-serial:988e9031364837354b:forward:tcp:0;localabstract:chrome_devtools_remote
[Chromedriver] [STDERR] [1568410030.672][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am force-stop com.android.chrome
[Chromedriver] [STDERR] [1568410030.802][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410030.802][INFO]: [366a898d88c48a013b901f4cd637640a] RESPONSE InitSession ERROR unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver] [STDERR] [1568410030.802][DEBUG]: Log type 'driver' lost 0 entries on destruction
[Chromedriver] [STDERR] [1568410030.802][DEBUG]: Log type 'browser' lost 0 entries on destruction
[debug] [WD Proxy] Got response with status 200: {"sessionId":"366a898d88c48a013b901f4cd637640a","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"366a898d88c48a013b901f4cd637640a","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),pl...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver]   (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[debug] [WD Proxy] Matched '/session' to command name 'createSession'
[debug] [WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"desiredCapabilities":{"chromeOptions":{"androidPackage":"com.android.chrome","androidDeviceSerial":"988e9031364837354b"},"loggingPrefs":{"browser":"ALL"}}}
[Chromedriver] [STDERR] [1568410031.060][INFO]: [db260142541c05c6225c81d45a9c89db] COMMAND InitSession {
[Chromedriver] [STDERR]    "desiredCapabilities": {
[Chromedriver] [STDERR]       "chromeOptions": {
[Chromedriver] [STDERR]          "androidDeviceSerial": "988e9031364837354b",
[Chromedriver] [STDERR]          "androidPackage": "com.android.chrome"
[Chromedriver] [STDERR]       },
[Chromedriver] [STDERR]       "loggingPrefs": {
[Chromedriver] [STDERR]          "browser": "ALL"
[Chromedriver] [STDERR]       }
[Chromedriver] [STDERR]    }
[Chromedriver] [STDERR] }
[Chromedriver] [STDERR] [1568410031.061][DEBUG]: Sending adb command: host:devices
[Chromedriver] [STDERR] [1568410031.062][DEBUG]: Received adb response: 988e9031364837354b      device
[Chromedriver] [STDERR] [1568410031.062][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm path com.android.chrome
[Chromedriver] [STDERR] [1568410031.526][DEBUG]: Received adb response: package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/base.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_autofill_assistant.config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_config.en.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.apk
[Chromedriver] [STDERR] package:/data/app/com.android.chrome-inmKYTHm9bFdW5JkNjX7KA==/split_vr.config.en.apk
[Chromedriver] [STDERR] [1568410031.526][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am set-debug-app --persistent com.android.chrome
[Chromedriver] [STDERR] [1568410031.575][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410031.575][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:pm clear com.android.chrome
[Chromedriver] [STDERR] [1568410032.133][DEBUG]: Received adb response: Success
[Chromedriver] [STDERR] [1568410032.133][DEBUG]: Sending command line file: /data/local/tmp/chrome-command-line
[Chromedriver] [STDERR] [1568410032.142][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am start -W -n com.android.chrome/com.google.android.apps.chrome.Main -d data:,
[Chromedriver] [STDERR] [1568410032.854][DEBUG]: Received adb response: Starting: Intent { dat=data:, cmp=com.android.chrome/com.google.android.apps.chrome.Main }
[Chromedriver] [STDERR] Status: ok
[Chromedriver] [STDERR] Activity: com.android.chrome/com.google.android.apps.chrome.Main
[Chromedriver] [STDERR] ThisTime: 575
[Chromedriver] [STDERR] TotalTime: 575
[Chromedriver] [STDERR] WaitTime: 627
[Chromedriver] [STDERR] Complete
[Chromedriver] [STDERR] [1568410032.854][DEBUG]: Sending adb command: host-serial:988e9031364837354b:forward:tcp:0;localabstract:chrome_devtools_remote
[Chromedriver] [STDERR] [1568410032.855][DEBUG]: Sending adb command: host:transport:988e9031364837354b|shell:am force-stop com.android.chrome
[Chromedriver] [STDERR] [1568410033.005][DEBUG]: Received adb response:
[Chromedriver] [STDERR] [1568410033.006][INFO]: [db260142541c05c6225c81d45a9c89db] RESPONSE InitSession ERROR unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver] [STDERR] [1568410033.006][DEBUG]: Log type 'driver' lost 0 entries on destruction
[Chromedriver] [STDERR] [1568410033.006][DEBUG]: Log type 'browser' lost 0 entries on destruction
[debug] [WD Proxy] Got response with status 200: {"sessionId":"db260142541c05c6225c81d45a9c89db","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)"}}
[WD Proxy] Got an unexpected response: {"sessionId":"db260142541c05c6225c81d45a9c89db","status":13,"value":{"message":"unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.\n  (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),pl...
[debug] [MJSONWP] Matched JSONWP error code 13 to UnknownError
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver]   (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:493:13)
[Chromedriver] Chromedriver exited unexpectedly with code null, signal SIGTERM
[debug] [Chromedriver] Changed state to 'stopped'
[Chromedriver] Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[Chromedriver]   (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[Chromedriver]     at Chromedriver.errorAndThrow [as start] (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:441:11)
[debug] [UiAutomator2] Deleting UiAutomator2 session
[debug] [UiAutomator2] Deleting UiAutomator2 server session
[debug] [WD Proxy] Matched '/' to command name 'deleteSession'
[debug] [WD Proxy] Proxying [DELETE /] to [DELETE http://localhost:8200/wd/hub/session/7615fb84-3cf9-4571-a0c0-ee01f6226621] with no body
[debug] [WD Proxy] Got response with status 200: "{\"sessionId\":\"7615fb84-3cf9-4571-a0c0-ee01f6226621\",\"status\":0,\"value\":\"Session deleted\"}"
[debug] [UiAutomator2] Resetting IME to 'com.sec.android.inputmethod/.SamsungKeypad'
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b shell ime set com.sec.android.inputmethod/.SamsungKeypad'
[debug] [Instrumentation] .
[debug] [Logcat] Stopping logcat capture
[debug] [ADB] Removing forwarded port socket connection: 8200
[debug] [ADB] Running 'C:\Users\akamble\AppData\Local\Android\Sdk\platform-tools\adb.exe -P 5037 -s 988e9031364837354b forward --remove tcp\:8200'
[debug] [BaseDriver] Event 'newSessionStarted' logged at 1568410033999 (17:27:13 GMT-0400 (Eastern Daylight Time))
[debug] [W3C] Encountered internal error running command: Error: Failed to start Chromedriver session: An unknown server-side error occurred while processing the command. Original error: unknown error: Failed to forward ports to device 988e9031364837354b: . unknown error: The adb command failed. Extra response: <>.
[debug] [W3C]   (Driver info: chromedriver=2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90),platform=Windows NT 10.0.17763 x86_64)
[debug] [W3C]     at Object.wrappedLogger.errorAndThrow (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-support\lib\logging.js:78:13)
[debug] [W3C]     at Chromedriver.errorAndThrow [as start] (C:\Users\swdl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:441:11)
[HTTP] <-- POST /wd/hub/session 500 26586 ms - 1309
[HTTP]
[debug] [Instrumentation] Time: 13.116
[debug] [Instrumentation]
[debug] [Instrumentation] OK (1 test)
[debug] [Instrumentation] The process has exited with code 0

## Code To Reproduce Issue [ Good To Have ]
Using  following capabilities to connect the Android chrome driver.
DesiredCapabilities capabilities = new DesiredCapabilities();

			capabilities.setCapability("deviceName", devicename);
			//capabilities.setCapability("udid", udid);
			capabilities.setCapability("platformName", environment);
			capabilities.setCapability("platformVersion", version);
			capabilities.setCapability("browserName", "Chrome");
			//capabilities.setCapability(MobileCapabilityType.BROWSER_NAME, "Chrome");
			capabilities.setCapability("noReset", true);
			capabilities.setCapability("automationName", "UiAutomator2");
			//capabilities.setCapability("forceMjsonwp", true);
			capabilities.setCapability("autoAcceptAlerts", true);
			capabilities.setCapability("autoGrantPermissions", true);
			capabilities.setCapability("unicodeKeyboard", true);
			capabilities.setCapability("resetKeyboard", true);
			capabilities.setCapability("showChromedriverLog", true);

			driver = new AndroidDriver(new URL("http://0.0.0.0:4723/wd/hub"), capabilities);

**Comment by mykola-mokhnach** [comment]
Duplicate of https://github.com/appium/appium/issues/13231

**Comment by AmitKambleOnset** [comment]
As per 13231 - I added chromedriverExecutable capability but still getting this error - unknown error: Failed to forward ports to device 1115fbf2b70c1003: . unknown error: The adb command failed. Extra response: <>.

**Comment by AmitKambleOnset** [comment]
@mykola-mokhnach  Thank you for giving the bug reference , Updated the platform tools to latest version(29.0.4) and it worked.
